test/mock-components/mock-period.js

```diff
@@ -0,0 +1 @@
+module.exports = 'October to December 2022'
```

Comment:
So our mock settlement is early Feb, does this period reflect what tht would look lik in the wild for an SFI case?

Reply:
Fair point.  Start month should be the month of the due date for Q1.  So changed this to December to February to match